lib/ansible/modules/windows/win_chocolatey.py

```diff
@@ -195,6 +195,8 @@
     - Specific version of the package to be installed.
     - When I(state) is set to C(absent), will uninstall the specific version
       otherwise all versions of that package will be removed.
+    - If lower version of package than expected is installed, I(state) must be C(latest)
```

Comment:
Might be better to say if a different version is already installed instead of just lower version?

Reply:
Agreed, I'll update!

Reply:
@jborean93 fixed 👍